README.md

```diff
@@ -14,6 +14,18 @@ This installs the module from npm and adds it as a dependency to the application
 
 If you create a MongoDB data source using the data source generator as described below, you don't have to do this, since the generator will run `npm install` for you.
 
+## Supported versions
+
+**Starting from the version 6.0.0, this connector is no longer compatible with LoopBack 3. Please use the latest 5.x version in your LoopBack 3 applications.**
+
+This module adopts the [Module Long Term Support (LTS)](http://github.com/CloudNativeJS/ModuleLTS) policy, with the following End Of Life (EOL) dates:
+
+| Version    | Status               | Published | EOL                  | LoopBack | Juggler  |
+| ---------- | -------------------- | --------- | -------------------- | ---------|----------|
+| 6.x        | Current              | Nov 2020  | Apr 2023 _(minimum)_ | 4        | 4.x      |
+| 5.x        | Active LTS           | Jun 2019  | Apr 2023             | 3, 4     | 3.x, 4.x |
+| 4.x        | Maintenance LTS      | Nov 2018  | Apr 2021             | 3, 4      | 3.x, 4.x  |
```

Comment:
Nipick - remove one space after `3, 4` in the last row to keep the the very last table cell indented with the rest.

```suggestion
| 6.x        | Current              | Nov 2020  | Apr 2023 _(minimum)_ | 4        | 4.x      |
| 5.x        | Active LTS           | Jun 2019  | Apr 2023             | 3, 4     | 3.x, 4.x |
| 4.x        | Maintenance LTS      | Nov 2018  | Apr 2021             | 3, 4     | 3.x, 4.x  |
```